cmd/strelaypoolsrv/gui/index.html

```diff
@@ -38,7 +45,7 @@
     <div class="container">
       <h1>Relay Pool Data</h1>
       <div ng-if="relays === undefined" class="text-center">
-        <img src="//cdnjs.cloudflare.com/ajax/libs/galleriffic/2.0.1/css/loader.gif" alt=""/>
+        <img src="https://cdnjs.cloudflare.com/ajax/libs/galleriffic/2.0.1/css/loader.gif" alt=""/>
```

Comment:
needs https removed

Reply:
Does it though? I think the root cause here is that it's difficult to test these pages without a fully fledged strelaypoolserv/ursrv setup running, so they're opened as files and then the `//...` url's don't work. I see no harm in hardcoding https for all of these

Reply:
Yes I have the same problem. For resolve there I had https for all of there